### DISCUSSION TITLE
Scaling of interfacial currents with particle shape

### DISCUSSION THREAD

**Discussion by dion-w** [pybamm-team/PyBaMM#1769] [Q&A] [answered]
hello everyone,
i have a question concerning the submodel "particle shape" : "user". I know that with this parameter you can reduce the amount of surface of your active material. For example if the particle shape is non-spherical or if some parts are blocked by the conductive binder fraction.

What i wonder: I tried some simulations with the user-supplied particle shape. Normally when you supply a very low current to your cell you get almost the theorethical capacity out of your cell, but this is not the case here. For a setup with an electrode capacity of 6mAh/cm² (positive electrode thickness ~100 µm) i almost loose 2 mAh/cm² at C/10 when using a user-supplied particle shape instead of the standard 3 e_s / R.

I think this is due to incorrectly scaling the molar flux in the microscale. Normally (and for this i refer to the [supporting_information.pdf](https://github.com/pybamm-team/PyBaMM/files/7448901/am8b08993_si_001.pdf) of [this paper](https://pubs.acs.org/doi/abs/10.1021/acsami.7b17771)) when you use a user-supplied surface area  you only set the surface area of the macroscale to this specific value. To ensure **charge conservation** the molar flux in the boundary needs to be scaled withe the factor of user-supplied-particle shape divided by 3*eps_s/R (see **equation S22 in the attached supporting information**).

Nonetheless i don't know how to fix this in the code. I only have an idea so far which i will show for the positive electrode (same for negative electrode):

- _pybamm/parameters/lithium_ion_parameters.py_: In method _set_scales() starting at Line 532, introduce not only a_p_typ and but also a blockage quotient (maybe called "a_block") being calculated <img src="https://render.githubusercontent.com/render/math?math=a_{block} = \frac{a_{p,typ}}{a_0}"> with <img src="https://render.githubusercontent.com/render/math?math=a_0 = \frac{3*{\epsilon}_{s,p}}{R_{p,typ}}"> . 

- _pybamm/models/submodels/interface/kinetics/butler_volmer.py_: <img src="https://render.githubusercontent.com/render/math?math=a_{block}"> could be used here in line 35 as an additional prefactor. Maybe it can used the same in other _get_kinetics methods of the interface subomdell, i don't know.

There may be a better name than <img src="https://render.githubusercontent.com/render/math?math=a_{block}"> since if <img src="https://render.githubusercontent.com/render/math?math=a_{block}"> is 1, no current is blocked but i didn't come up with a better name so far.

So what do you think about it?

**Comment by valentinsulzer** [comment]
I think this sounds sensible. `a_block` should probably also go into the SEI and plating kinetics?

**Reply by dion-w** [reply]
> 
> 
> I think this sounds sensible. `a_block` should probably also go into the SEI and plating kinetics?

Would have to discuss this with other members of my institute since i'm not all involved in SEI or plating kinetics in P2D-models.

**Comment by valentinsulzer** [comment, marked as answer]
See #2254

**Reply by dion-w** [reply]
I think this has been solved by using the utilisation that impacts the kinetics right? I don't know the specific issue or pull request number though.

**Reply by valentinsulzer** [reply]
Yes,  #1821
